activesupport/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 *   Add `String#upcase_first` method.
 
-     *Glauco Custódio*
+    *Glauco Custódio*, *bogdanvlviv*
```

Comment:
@rafaelfranca, https://github.com/rails/rails/blob/master/guides/source/contributing_to_ruby_on_rails.md#updating-the-changelog 
"You should add an entry to the top of the CHANGELOG of the framework that you modified if you're adding or removing a feature, committing a **bug fix** or adding deprecation notices. Refactorings and documentation changes generally should not go to the CHANGELOG." 
I have done fix bug.